projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -48,8 +48,9 @@ public function init() {
 			return;
 		}
 
+		$in_jetpack = ( class_exists( 'Jetpack' ) && method_exists( 'Jetpack', 'enable_module_configurable' ) ) ? true : false;
```

Comment:
I think we're using this logic in a few places. If we need to do that, would it be worth creating a helper function somewhere? I can see that we might be 'in Jetpack' under other circumstances, like in the standalone plugin, and we may have to update the logic.

Reply:
We also probably don't need to make it a ternary, just `class_exists( 'Jetpack' ) && method_exists( 'Jetpack', 'enable_module_configurable' )` should suffice.

Reply:
I will get rid of the ternary, but I actually ended up removing the extra `in_jetpack` checks.